src/Models/IoTContainerizedProject.ts

```diff
@@ -147,10 +147,21 @@ export class IoTContainerizedProject extends IoTWorkbenchProjectBase {
           `Internal Error. Could not find iot workbench project file.`);
     }
 
-    // Open project
     await this.openProject(this.projectRootPath, openInNewWindow);
   }
 
+  async openFolderInContainer(folderPath: string) {
+    if (!await FileUtility.directoryExists(ScaffoldType.Local, folderPath)) {
+      throw new Error(
+          `Fail to open folder in container. ${folderPath} does not exist.`);
+    }
+
+    setTimeout(
+        () => vscode.commands.executeCommand(
+            'remote-containers.openFolder', vscode.Uri.file(folderPath)),

```

Comment:
Since Cube extension is weakly dependent, what if remote extension is not installed here?

Reply:
Fixed.